genai-perf/tests/test_json_exporter.py

```diff
@@ -682,3 +682,399 @@ def test_invalid_goodput_json_output(
         assert output_data_dict["input_config"]["goodput"] == json.loads(
             expected_invalid_goodput_json_config
         )
+
+    def test_triton_telemetry_output_success(
+        self, monkeypatch, mock_read_write: pytest.MonkeyPatch
+    ) -> None:
+        stats = {
```

Comment:
this seems to be reused over and over in each tests. can we take it out using `pytest.fixture`?

Reply:
I didn't see that the stats is a class-level member. So, I re-organized and used it.